### PR TITLE
Updated script to download Transifex translations

### PR DESCRIPTION
**Pull request by calmh**
Throw away our old API integration and use the `tx` CLI, via a docker container.

### CODE REVIEW

**Comment by acolomb** [comment]
No time to review anymore today, but will try to get to it tomorrow.

**Comment by acolomb** [comment]
Are you planning to reimplement the `prettyprint.js` and `valid-langs.js` stuff again?

With Weblate, I was going to stick closely to what the old script did, which is rather easy with their API. The main difference will be that we should keep all translations (whether valid or not) in our Git repo, as Weblate basically uses Git as its storage and preferably everything is synced with upstream. So filtering should still happen through the `valid-langs.js` file, regardless of how many JSON translations are sitting around.

What reasoning do we have for rewriting underscores to hyphens in the language codes? With both solutions, just keeping the underscore would be easier, no?

**Comment by calmh** [comment]
Right, those still need generating... The reason for dashes is that it's what's expected by the GUI framework, afaik.

File: script/transifexdl.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# in $TRANSIFEX_TOKEN.
+docker run -it --rm \
+    -v $(pwd):/src -w /src \
+    -v /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt \
+    -e TX_TOKEN=$TRANSIFEX_TOKEN \
+    transifex/txcli pull -a -f
+
+# Rename ab_CD language codes to ab-CD
+pushd gui/default/assets/lang
+for f in *_* ; do
```
Comment (acolomb) [inline script/transifexdl.sh:16]:
```suggestion
for f in lang-*_*.json ; do
```

**Review by acolomb — CHANGES_REQUESTED** [review]

**Comment by calmh** [comment]
Now irrelevant

**Pull request closed without merging**